tracks/ruby/exercises/isogram/mentoring.md

```diff
@@ -1,12 +1,27 @@
 ### Reasonable solutions
 
+Using `uniq`
 ```ruby
 def isogram?(string)
-  letters = string.downcase.scan(/[a-z]/)
+  letters = string.downcase.scan(/[[:alpha:]]/)
```

Comment:
Since it is downcased, perhaps the more explicit search for lowercase letters makes sense.  This expands the search to twice as many characters without benefit.

Reply:
True, but `[a-z]` cover letters only from English alphabet comparing to `[[:alpha:]]`
http://www.rubular.com/r/x6xKwSkTSl
It's out of description, but why not to have it in place?

Reply:
In that case, it may be worthwhile contributing to the specifications repository to get the exercise changed.

Reply:
All exercises in exercism always ever cover only 7 bit ASCII.

Unless the track specifies extra unicode tests, I do not think that any unicode specialties need to be covered.

Using `[[:alpha:]]` will bloat the resulting DFA without any need.

But as we are not necissarily optimising for speed here, I have to admit, I consider `[[:alpha:]]` more readable than `[a-z]` or `\pLl` (or whatever lowercase letter group was).